src/Draco.Compiler/Internal/Symbols/Symbol.cs

```diff
@@ -105,9 +142,14 @@ public virtual string FullName
     public virtual IEnumerable<Symbol> InstanceMembers => this.Members.Where(x => x is IMemberSymbol mem && !mem.IsStatic);
 
     /// <summary>
-    /// Documentation attached to this symbol.
+    /// The raw documentation attached to this symbol.
+    /// </summary>
+    public virtual string RawDocumentation => string.Empty;
```

Comment:
Maybe we want to remove raw doc as a whole. I don't see a reason to expose it, when we have the structured version.